### PR TITLE
ignore some errors at module level

### PR DESCRIPTION
**Pull request by lucascolley**
closes gh-561

cc @stefanv

### CODE REVIEW

File: numpydoc/__main__.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 """
-Implementing `python -m numpydoc` functionality.
+Implementing `python -m numpydoc` functionality
```
Comment (larsoner) [inline numpydoc/__main__.py:2]: I think all docstrings (including module-level?) typically end with a period, no? In that sense this is a tiny regression. If it's for testing purposes (looks like it?) then a little comment here saying that's the case would help the next person not (mistakenly) fix it.

**Review by larsoner — APPROVED** [review]
Just one tiny comment from me.

@stefanv did you want to look?

File: numpydoc/__main__.py (hunk 1 above)
Comment (lucascolley) [reply numpydoc/__main__.py:2]: Yeah, I can add a comment. Also happy to change to a different "defect" if we want.

**Comment by lucascolley** [comment]
thanks @larsoner, addressed!

**Review by stefanv — APPROVED** [review]

**Comment by stefanv** [comment]
Thanks, Lucas!

**Pull request merged**